### PR TITLE
[disqus][l]: added disqus plugin

### PR DESCRIPTION
**Pull request by svetozarstojkovic**
PR created per https://gitlab.com/datopian/ckan-next-gen/issues/61

### CODE REVIEW

File: plugins/disqus/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
```
Comment (anuveyatsu) [inline plugins/disqus/index.js]: Where are you using `fs`?

File: plugins/disqus/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const path = require('path')
```
Comment (anuveyatsu) [inline plugins/disqus/index.js]: same

File: plugins/disqus/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const fetch = require('node-fetch')
```
Comment (anuveyatsu) [inline plugins/disqus/index.js]: same

File: plugins/disqus/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+const config = require('../../config')
+let configApiUrl = config.get("API_URL")
```
Comment (anuveyatsu) [inline plugins/disqus/index.js]: same

File: plugins/disqus/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = function (app) {
+  app.use((req, res, next) => {
+    let disqusPages = "," + config.get('DISQUS_PAGES') + ",";
+    let currentUrl = "," + req.url + ",";
+
+    res.locals.disqusEnabled = disqusPages.includes(currentUrl);
+    res.locals.PAGE_URL =
```
Comment (anuveyatsu) [inline plugins/disqus/index.js]: doesn't it throw an error?

File: README.md [hunk 6]
```diff
@@ -268,6 +268,16 @@ Here is the list of well-known services that can be used without setting host an
 
 Then you need to implement `contact.html` template in your theme so that a contact form can be rendered at `/contact`.
 
+#### Disqus plugin
```
Comment (anuveyatsu) [inline README.md]: Can we start having `README.md` within a plugin directory? See the example of `wp` plugin please.

**Comment by starsinmypockets** [comment]
Closed: This is implemented in the Montreal theme for now. I don't think this is ready / useful to be implemented as a plugin since the disqus block needs to be added to the base.html.

**Pull request closed without merging**